test/visualization/test_visualization.py

```diff
@@ -74,7 +74,8 @@ def test_acceptance_rates_trajectory():
         histories, labels, yscale='log', rotation=76)
     _, ax = plt.subplots()
     pyabc.visualization.plot_acceptance_rates_trajectory(
-        histories, labels, yscale='log10', rotation=76, size=(10, 5), ax=ax)
+        histories, labels, yscale='log10', rotation=76, size=(10, 5), ax=ax
+        normalize_by_ess=True)
```

Comment:
One last thing: Could you once run this with normalize_by_ess=False, and once with True? So as to test both modes.

Reply:
Done!